include/z64item.h

```diff
@@ -283,24 +283,24 @@ typedef enum {
     /* 0x87 */ ITEM_RUPEE_PURPLE,
     /* 0x88 */ ITEM_RUPEE_GOLD,
     /* 0x89 */ ITEM_INVALID_8,
-    /* 0x8A */ ITEM_STICKS_5,
-    /* 0x8B */ ITEM_STICKS_10,
-    /* 0x8C */ ITEM_NUTS_5,
-    /* 0x8D */ ITEM_NUTS_10,
+    /* 0x8A */ ITEM_DEKU_STICKS_5,
+    /* 0x8B */ ITEM_DEKU_STICKS_10,
+    /* 0x8C */ ITEM_DEKU_NUTS_5,
+    /* 0x8D */ ITEM_DEKU_NUTS_10,
     /* 0x8E */ ITEM_BOMBS_5,
     /* 0x8F */ ITEM_BOMBS_10,
     /* 0x90 */ ITEM_BOMBS_20,
     /* 0x91 */ ITEM_BOMBS_30,
-    /* 0x92 */ ITEM_ARROWS_SMALL,
-    /* 0x93 */ ITEM_ARROWS_MEDIUM,
-    /* 0x94 */ ITEM_ARROWS_LARGE,
-    /* 0x95 */ ITEM_SEEDS_30,
+    /* 0x92 */ ITEM_ARROWS_5,
+    /* 0x93 */ ITEM_ARROWS_10,
+    /* 0x94 */ ITEM_ARROWS_30,
+    /* 0x95 */ ITEM_DEKU_SEEDS_30,
     /* 0x96 */ ITEM_BOMBCHUS_5,
     /* 0x97 */ ITEM_BOMBCHUS_20,
-    /* 0x98 */ ITEM_STICK_UPGRADE_20,
-    /* 0x99 */ ITEM_STICK_UPGRADE_30,
-    /* 0x9A */ ITEM_NUT_UPGRADE_30,
-    /* 0x9B */ ITEM_NUT_UPGRADE_40,
+    /* 0x98 */ ITEM_DEKU_STICK_UPGRADE_20,
+    /* 0x99 */ ITEM_DEKU_STICK_UPGRADE_30,
+    /* 0x9A */ ITEM_DEKU_NUT_UPGRADE_30,
+    /* 0x9B */ ITEM_DEKU_NUT_UPGRADE_40,
```

Comment:
Maybe this should be `CAPACITY` instead of `UPGRADE`. And the same for the quivers/bomb bags: it's not 30/40/50 of them!

Reply:
plenty of other things rely on this upgrade naming scheme, so it wouldnt be so simple to change its name. `UpgradeType` enum, `Inventory_ChangeUpgrade`, `gSaveContext.inventory.upgrades` etc

Reply:
well since `ITEM_BOMB_BAG_40` doesn't have "upgrade" why not have `ITEM_DEKU_NUT_CAPACITY_40` in theory, but I don't think it's much of an improvement

idk I don't think we can have something truly satisfying on the "upgrades items", and current master looks okay on that front

Reply:
Is this considered resolved?
or can we at least leave this to another PR/discussion (that I wouldn't open/start, feel free to do so Elliptic though)

Reply:
Bump: Is this considered resolved?
I'll assume yes in like 7 days or so